buildchain/buildchain/targets/package.py

```diff
@@ -238,9 +239,13 @@ def build_srpm(self) -> types.TaskDict:
     def _download_sources(self) -> None:
```

Comment:
Function should be renamed (`_import_sources`? `_get_sources`?) and docstring updated.